dist/docs/guide/usage/install.adoc

```diff
@@ -0,0 +1,89 @@
+### Installation
+
+To provision a Wildfly server, Prospero requires following information:
+
+* manifest definition
+* location of channel's Maven repositories
+* feature-pack name
+
```

Comment:
We should maybe state that there are three possible inputs for the installation: 1. predefined profile; 2. fpl; 3. galleon provisioning.xml file, and give examples for each of those. It would then come up naturally that provisioning.xml allows for customizations.

Now in the "Customizing server installation" section bellow the provisioning.xml file comes bit out of the blue.

Reply:
Good point. I think I'm going to create 3 sub-sections describing each of the different options:

 * Installing predefined server profile
 * Installing a feature pack
 * Installing a customized feature pack

Reply:
+1